src/rules/lua-truthiness.ts

```diff
@@ -9,7 +9,6 @@ export const luaTruthiness = makeRule<[], "falsyStringNumberCheck">({
 		type: "problem",
 		docs: {
 			description: "Warns against falsy strings and numbers",
-			recommended: false,
```

Comment:
Is it possible to be explicit in this field, or does the new string option only allow "recommended" or <absent>?

Reply:
Yes. Recommended or absent are the only options here.
![2024-05-04_13-26](https://github.com/roblox-ts/eslint-plugin-roblox-ts/assets/32301681/bb75a680-0939-40e0-a464-6f001cfb98f0)